### ISSUE TITLE
How to segment bodies within the cell

### ISSUE THREAD

**Issue by kayleemisuraca** [MouseLand/cellpose#274]
Is it possible to segment bodies within a cell? My data is a set of TEM images. I am looking to segment autophagic bodies inside the vacuole of the cell. When I tried it on my images, it highlighted the vacuole but left out the autophagic bodies within the vacuole. I attached a picture of the bodies and vacuole I'm hoping Cellpose can highlight. 

If anybody has used Cellpose to segment organelles that would also be helpful here.

Thanks!
![safe example for forum](https://user-images.githubusercontent.com/85602088/121295292-c5086280-c8bc-11eb-8499-4ec04798c7a8.jpg)
### ****

**Comment by kevinjohncutler** [comment]
I am guessing this should in principle be quite similar to nuclear segmentation, although in that case usually the nucleus is stained in a different channel and segmented separate from the cytosol.  Are those dark lines drawn in or part of the original image? 

Cellpose doesn't handle nested cells right now, but a workaround is that you could train one model to detect the vacuole, and then train another model to only detect the autophagic bodies - so two different ground truth masks per image.

**Comment by kayleemisuraca** [comment]
The dark lines are drawn in. Ideally, we want our output image to have what is inside those dark outlines shaded in. This wouldn't work with the pretrained model? One of our obstacles is our small dataset, so I was hoping to use the pretrained model to solve that issue.

**Comment by saskra** [comment]
I used Cellpose on mitochondria measured by electron microscopy, which worked reasonably well, so round organelles should be segmentable in principle. However, the quality was not good enough for us with the pre-trained model, so I plan to re-train on our own data in the future.

**Comment by carsen-stringer** [comment]
very cool, @saskra , if you train and share the model we can link to it in the documentation. but yeah if it doesn't work well when you test it on https://www.cellpose.org/ then you likely need to train on new images

it may be a good idea to have these discussions on imagesc too https://forum.image.sc/tag/cellpose

I'm going to close this for now as this isn't necessarily a code issue
